src/test/java/org/jpeek/graph/DisjointTest.java

```diff
@@ -38,7 +37,6 @@
 public final class DisjointTest {
```

Comment:
@HDouss Could you please add one more test where the nodes in the graph are not connected at all?